src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz.app.src

```diff
@@ -23,6 +23,7 @@
   {applications, [kernel,
                   stdlib,
                   stats_hero,
+                  chef_objects,
```

Comment:
Could you please add more context related to this?

Reply:
There's context in the commit.  Would you rather have the context in the code itself?
```
    ===> Compiling oc_chef_authz
    ===> Compiling apps/oc_chef_authz/src/oc_chef_policy_group.erl failed
    apps/oc_chef_authz/src/oc_chef_policy_group.erl:26:2: behaviour chef_object undefined
```

Reply:
Is it the same before erlang upgarde 24 ?

Reply:
@jashaik No errors before the erlang 24 upgrade.

Reply:
does this make sense to be included as a part of the Erlang bump PR then?